helm/cluster-azure/values.yaml

```diff
@@ -1,3 +1,4 @@
+baseDomain: azuretest.gigantic.io
```

Comment:
Values are now managed `from the schema` and then generated with `make ...` 

see the readme

Reply:
essentially just add the `default value` to `baseDomain` in the schema and run `values-generate`